common-npm-packages/docker-common-v2/registryauthenticationprovider/acrauthenticationtokenprovider.ts

```diff
@@ -1,8 +1,11 @@
 "use strict";
 
+import AuthenticationTokenProvider from "./authenticationtokenprovider";
+import { ApplicationTokenCredentials } from "azure-pipelines-tasks-azure-arm-rest-v2/azure-arm-common";
```

Comment:
For my understanding - When do you use import {ApplicationTokenCredentials} from "./" over import ApplicationTokenCredentials from "./" without braces?

A